eli5/explain.py

```diff
@@ -106,6 +106,14 @@ def explain_prediction(estimator, doc, **kwargs):
 
         This argument may be supported or not, depending on estimator type.
 
+    top_targets : int, optional
+        Number of targets to show. When ``top_targets`` is provided,
+        only specified number of targets with highest scores are shown.
```

Comment:
What do you think about supporting negative top_targets? E.g. -1 would mean 1 target with lowest score. I'm not sure it is important for classification, but it could be useful for regression.